onebusaway-android/src/main/java/com/joulespersecond/oba/ObaAnalytics.java

```diff
@@ -103,6 +103,35 @@ public String toString() {
      * @param label    label name
      */
     public static void reportEventWithCategory(String category, String action, String label) {
+        if (isAnalyticsActive()) {
+            Tracker tracker = Application.get().getTracker(Application.TrackerName.APP_TRACKER);
+            Tracker tracker2 = Application.get().getTracker(Application.TrackerName.GLOBAL_TRACKER);
+            String obaRegionName = getObaRegionName();
+
+            tracker.send(new HitBuilders.EventBuilder()
+                    .setCategory(category)
+                    .setAction(action)
+                    .setLabel(label)
+                    .setCustomDimension(1, obaRegionName)
+                    .build());
+            tracker2.send(new HitBuilders.EventBuilder()
+                    .setCategory(category)
+                    .setAction(action)
+                    .setLabel(label)
+                    .setCustomDimension(1, obaRegionName)
+                    .build());
+        }
+    }
+
+    /**
+     * Reports events with categories even when GA is disabled.
+     * Helps segmentation in GA admin console.
+     *
+     * @param category category name
+     * @param action   action name
+     * @param label    label name
+     */
+    public static void forceReportEventWithCategory(String category, String action, String label) {
```

Comment:
I'd prefer not to expose a public static method that allows us to bypass the user's preference for not reporting analytics.  It's too easy for someone else who isn't as familiar with the code base to come along and accidentally start sending data via this method without checking the preference.  I'd prefer to have a private method in PreferencesActivity handle this, and to have it well commented so others know that the only time this should be used is when we're disabling GA reporting, and reporting that disabling event via GA.  See also my comment in PreferencesActivity.